src/interfaces.ts

```diff
@@ -0,0 +1,37 @@
+export interface ErrorParams {
+  firstName?: string;
+  lastName?: string;
+  birthDate?: string;
+  email?: string;
+  "home-line1"?: string;
+  "home-line2"?: string;
+  "home-city"?: string;
+  "home-state"?: string;
+  "home-zip"?: string;
+  "work-line1"?: string;
+  "work-line2"?: string;
+  "work-city"?: string;
+  "work-state"?: string;
+  "work-zip"?: string;
```

Comment:
How come some of these are in quotes?

Reply:
You can't have `work-zip` as a key in an object so it must be in quotes.